content/tutorial-nlp-from-scratch.md

```diff
@@ -0,0 +1,848 @@
+
++++
+
+Today, Deep Learning is getting adopted in everyday life and now it is more important to ensure that decisions that have been taken using AI are not reflecting discriminatory behavior towards a set of populations. It is important to take fairness into consideration while consuming the output from AI. Throughout the tutorial we'll try to question all the steps in our pipeline from an ethics point of view.
+
++++
+
+## Prerequisites 
+
+You are expected to be familiar with the Python programming language and array manipulation with NumPy. In addition, some understanding of Linear Algebra and Calculus is recommended. You should also be familiar with how Neural Networks work. For reference, you can visit the [Python](https://docs.python.org/dev/tutorial/index.html), [Linear algebra on n-dimensional arrays](https://numpy.org/doc/stable/user/tutorial-svd.html) and [Calculus](https://d2l.ai/chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.html) tutorials.
+
+To get a refresher on Deep Learning basics, You should consider reading [the d2l.ai book](https://d2l.ai/chapter_recurrent-neural-networks/index.html), which is an interactive deep learning book with multi-framework code, math, and discussions. You can also go through the [Deep learning on MNIST from scratch tutorial](https://numpy.org/numpy-tutorials/content/tutorial-deep-learning-on-mnist.html) to understand how a basic neural network is implemented from scratch.
+
+In addition to NumPy, you will be utilizing the following Python standard modules for data loading and processing:
+- [`pandas`](https://pandas.pydata.org/docs/) for handling dataframes 
+- [`Matplotlib`](https://matplotlib.org/) for data visualization
+
+This tutorial can be run locally in an isolated environment, such as [Virtualenv](https://virtualenv.pypa.io/en/stable/) or [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html). You can use [Jupyter Notebook or JupyterLab](https://jupyter.org/install) to run each notebook cell.
+
++++
+
+## Table of contents
+
+1. Data Collection 
+
+2. Preprocess the datasets
+
+3. Build and train a LSTM network from scratch
+
+4. Perform sentiment analysis on collected speeches 
+
+5. Next steps
+
++++
+
+## 1. Data Collection
+
+Before you begin there are a few pointers you should always keep in mind before choosing the data you wish to train your model on:
+- **Identifying Data Bias** - Bias is a component of the human thought process, and data collected from humans therefore inherently reflects that bias. Some ways in which this bias tends to occur in Machine Learning datasets are:
```

Comment:
it was intended to be `from` because we're basically trying to convey that data sourced from human activities can reflect the bias in their minds. I'll try rephrasing it